packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -382,6 +404,7 @@ export default class ScriptTransformer {
           displayErrors: true,
           filename: isCoreModule ? 'jest-nodejs-core-' + filename : filename,
         }),
+        scriptContent: wrappedCode,
```

Comment:
I hope this won't increase memory footprint too much (thinking about some big scripts like TS main file)

Reply:
we can make sure to not return the `Script` if we haven't wrapped anything as it'll be broken aynway

Reply:
I've done that, although we'll need access to the pre-transformed source as well for v8 coverage. Can revisit in that PR though